Chandra.taco/test_random_salt.py

```diff
@@ -1,8 +1,9 @@
+from __future__ import print_function
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import numpy as np
 
 import Chandra.taco
-print Chandra.taco.__file__
+print(Chandra.taco.__file__)
```

Comment:
This could probably just be deleted.  It's leftover from when I wasn't confident about testing namespace packages.  They still drive me nuts but at least I am pretty sure about how they import.